lua-aws/services/base.lua

```diff
@@ -2,6 +2,28 @@ local class = require ('lua-aws.class')
 local API = require ('lua-aws.api')
 local util = require ('lua-aws.util')
 
+local specs = {}
+
+local function get_spec(aws, service_name)
+	local spec = specs[service_name]
+	if spec then
+		return spec
+	end
+
+	spec = {}
+
+	aws:fs().scandir(util.script_path() .. '/specs/', service_name, function (path)
+		local version = path:match('[^%-]*%-([%w%-]*)%.min%.json')
```

Comment:
It seems `:match()` works fine as far as I tested. additionally, changed `.js` to `.json`.

Reply:
oh, previously spec actually defined by javascript. but now matching `json` should be better. thanks!

Reply:
and `print` seems to disappear, I will merge this.